Cloudinary/Actions/DelResParams.cs

```diff
@@ -19,6 +20,17 @@ public DelResParams()
 
         public string Type { get; set; }
 
+        public SpecialImageType SpecialType {

```

Comment:
Where is SpecialImageType used? I see that in this commit "none" is used.
Sprite and Text are also considered as "Type" (not special)